src/Aspire.Hosting.Azure.AppContainers/AzureContainerAppsInfrastructure.cs

```diff
@@ -29,7 +29,7 @@ internal sealed class AzureContainerAppsInfrastructure(
 {
     public async Task BeforeStartAsync(DistributedApplicationModel appModel, CancellationToken cancellationToken = default)
     {
-        if (executionContext.IsRunMode)
+        if (!executionContext.IsPublishMode)
```

Comment:
I don't like this at all. Can we just not run any of these in inspect mode?

Reply:
I'll look at selectively disabling.